Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -187,5 +186,116 @@ def proximal(self, x, tau, out = None):
 
         return soft_shrinkage(x, tau, out = out) 
 
+class WeightedL1Norm(Function):
+    r"""WeightedL1Norm function
+            
+            .. math:: F(x) = ||x||_{\ell^1(w)} 
+            
+            
+    Where :math:`w` is array of positive weights
+    
+    Parameters:
+    -----------
+
+    weight: DataContainer, numpy ndarray, default None
+        Array of weights matching the size of the wavelet coefficients
```

Comment:
Remove the reference to wavelets here. 

To match the weightedL2NormSquared this could be 
weight: a `scalar` or a `DataContainer` with the same shape as the intended domain of this `WeightedL1Norm` function